modeler.js

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+// An action that retrieves a file.
+async function retrieveFile([fileId]) {
+    const modelerAudience = process.env.MODELER_AUDIENCE;
+    const accessToken = await getAccessToken("components", modelerAudience);
+
+    const url = `process.env.MODELER_AUDIENCE/files/${fileId}`;
+
+// Configure the API call.
+  const options = {
+    method: "GET",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+
+  try {
+    const response = await axios(options);
+    const results = response.data;
+
+    results.forEach(x => console.log(`File retrieved! ID: ${id}. Name: ${name}`));
```

Comment:
I think I mentioned this last week, but I'm just wondering how to format the output properly, and what we'd want to include.

Reply:
I think this makes sense. We basically want enough information to identify the item. The user might want to display different things, but we leave that up to them, and we just demonstrate that we're getting critical information about the items.

Reply:
I know that {name} isn't gonna work here, but let me know if you have any other suggestions 👍